libecl/tests/ecl_grid_cell_contains_wellpath.c

```diff
@@ -44,14 +44,13 @@ vector_type * load_expected( const char * filename ) {
   vector_type * expected = vector_alloc_new();
 
   while (true) {
-    double x,y,z,md;
+    double x,y,z;
     int i,j,k;
 
-    if (fscanf( stream , "%lg %lg %lg %lg %d %d %d" , &x,&y,&md,&z,&i,&j,&k) == 7) {
+    if (fscanf( stream , "%lg %lg %lg %d %d %d" , &x,&y,&z,&i,&j,&k) == 6) {
```

Comment:
has `md` been removed from the file or was it never there?  Isn't it a crucial part of a wellpath?